client/src/pages/EventsPage/EventsPage.jsx

```diff
@@ -1,11 +1,94 @@
+		description:
+			"Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+		category: "Surgical",
+	},
+	{
+		title: "Diagnostic Imaging",
+		image:
+			"https://s3-prod.modernhealthcare.com/s3fs-public/SPONSORED_170619878_AR_-1_RXMUPRMWBUGI.jpg",
+		description:
+			"Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+		category: "Diagnostic",
+	},
+];
 
 const EventsPage = () => {
+	const [searchValue, setSearchValue] = useState("");
+	const [selectedCategory, setSelectedCategory] = useState("All");
+
+	const handleSearch = (event) => {
+		setSearchValue(event.target.value);
+	};
+
+	const handleCategoryChange = (event) => {
+		setSelectedCategory(event.target.value);
+		setSearchValue("");
+	};
+
+	const filteredEvents = eventsData.filter((item) => {
+		const titleMatches = item.title
+			.toLowerCase()
+			.includes(searchValue.toLowerCase());
+		const contentMatches = item.description
+			.toLowerCase()
+			.includes(searchValue.toLowerCase());
+		const categoryMatches =
+			selectedCategory === "All" || item.category === selectedCategory;
+
+		return (titleMatches || contentMatches) && categoryMatches;
+	});
+
 	return (
 		<>
-			<h1>Events Page</h1>
+			<h1>Events</h1>
+			<Line />
+			<div className="tools-bar">
+				<input
+					type="text"
+					placeholder="Search"
+					value={searchValue}
+					onChange={handleSearch}
+				/>
+				<select value={selectedCategory} onChange={handleCategoryChange}>
+					<option value="All">All</option>
+					<option value="Emergency">Emergency</option>
+					<option value="Surgical">Surgical</option>
+					<option value="Diagnostic">Diagnostic</option>
+				</select>
+				<p>{filteredEvents.length} events found</p>
+			</div>
```

Comment:
for accessibility let's make this a form

```suggestion
			<form className="search-form">
				<input
					type="text"
					placeholder="Search"
					value={searchValue}
					onChange={handleSearch}
				/>
				<select value={selectedCategory} onChange={handleCategoryChange}>
					<option value="All">All</option>
					<option value="Emergency">Emergency</option>
					<option value="Surgical">Surgical</option>
					<option value="Diagnostic">Diagnostic</option>
				</select>
				<p>{filteredEvents.length} events found</p>
			</form>
```